public/app/features/sandbox/TestStuffPage.tsx

```diff
@@ -164,12 +157,12 @@ function LinkToBasicApp({ placement }: { placement: string }) {
 
   return (
     <div>
-      {extensions.map((extension) => {
+      {extensions.map((extension, i) => {
         if (!isPluginExtensionLink(extension)) {
           return null;
         }
         return (
-          <LinkButton href={extension.path} title={extension.description} key={extension.key}>
+          <LinkButton href={extension.path} title={extension.description} key={i}>
```

Comment:
We need to add the `key` or back. Using the index might cause weird behaviour in the UI since the extensions might return different values depending on the passed context.

But maybe we should call the `key` -> `id` instead.

```suggestion
          <LinkButton href={extension.path} title={extension.description} key={extension.key}>
```

Reply:
I have changed it to `id` (pushing it soon) and using almost the same hash-generating mechanism as we had before. Hopefully we can introduce extension ids in the configuration as well, soon, and then we can switch to using those instead of the hashes as well.